layers/generate_layers.py

```diff
@@ -173,6 +178,34 @@ def fixProps(inputProps):
 
     convertGeoJsonToPmtiles(f"{tmp}/mrn.geojson", "output/mrn.pmtiles")
 
+def makeNationalCycleNetwork():
+    tmp = "tmp_ncn"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    # Get the shapefile
+    run(
+        [
+            "wget",
+            "https://services5.arcgis.com/1ZHcUS1lwPTg4ms0/arcgis/rest/services/National_Cycle_Network_Public/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson",
```

Comment:
How'd you get this link? I got `https://opendata.arcgis.com/api/v3/datasets/5defd254e78745bfb12d0456abc1bcf1_0/downloads/data?format=geojson&spatialRefId=4326&where=1%3D1` by going to https://data-sustrans-uk.opendata.arcgis.com/datasets/5defd254e78745bfb12d0456abc1bcf1_0/explore, clicking download, going down to geojson, choosing the file that's already there

Reply:
Got the same link as you, will use it instead and rerun